pkg/master/services.go

```diff
@@ -25,12 +25,12 @@ import (
 	"k8s.io/kubernetes/pkg/registry/core/service/ipallocator"
 )
 
-// DefaultServiceIPRange takes a the serviceIPRange flag and returns the defaulted service ip range (if  needed),
+// ServiceIPRange takes a the serviceIPRange flag and returns the defaulted service ip range (if  needed),
```

Comment:
I noticed a couple of typos in the docs, s/takes a the/takes the/, and a double space in 'if  needed'.

Beyond the typos, perhaps the docs could be:

```
// ServiceIPRange checks if the serviceIPRange flag is nil, setting service ip
// range to the default value in kubeoptions.DefaultServiceIPCIDR in that case.
// Returns service ip range, api server service IP, and an error
```

Reply:
made the change. 
/retest